Research/ResearchUI/iOS/Views/RSDCountdownDial.swift

```diff
@@ -290,16 +304,23 @@ public final class RSDCountdownDial: RSDProgressIndicator, RSDViewDesignable {
         dialLayer?.fillColor = UIColor.clear.cgColor
     }
     
-    private func createCirclePath() -> UIBezierPath {
+    private func createCirclePath(isClockwise: Bool) -> UIBezierPath {
         
         let inset: CGFloat = 0.0
         let radius = (bounds.width - inset) / 2.0
         let arcCenter = CGPoint(x: bounds.width / 2.0, y: bounds.height / 2.0)
-        let startAngle = -1.0 * CGFloat.pi / 2.0
-        let endAngle = 3.0 * CGFloat.pi / 2.0
+        
+        var startAngle = -1.0 * CGFloat.pi / 2.0
+        var endAngle = 3.0 * CGFloat.pi / 2.0
+        
+        // Counter clock-wise requires different start/end angles
+        if !isClockwise {
+            startAngle = 3.0 * CGFloat.pi / 2.0
+            endAngle = -1.0 * CGFloat.pi / 2.0
+        }
```

Comment:
```
    private var minAngle: CGFloat { 
        return -1.0 * CGFloat.pi / 2.0 
    }

   private var maxAngle: CGFloat { 
        return 3.0 * CGFloat.pi / 2.0 
    }

    private func createCirclePath() -> UIBezierPath {
        let startAngle = isClockwise ? minAngle : maxAngle
        let endAngle = isClockwise ? maxAngle : minAngle
        let shapePath = UIBezierPath()
        shapePath.addArc(withCenter: arcCenter, radius: radius, startAngle: startAngle, endAngle: endAngle, clockwise: isClockwise)
        
         return shapePath
     }
```

I think this would be more maintainable. It allows for a certain degree of future proofing where `minAngle` and `maxAngle` may be exposed as overridable open vars. Additionally, It then means that both the ring and the dial are drawn in the same direction.

Unless there is some sort of weird performance hit that I am not seeing?